src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableStatementTest.java

```diff
@@ -50,6 +52,9 @@ public class CallableStatementTest extends AbstractTest {
     private static String getObjectLocalDateTimeProcedureName = RandomUtil.getIdentifier("CallableStatementTest_getObjectLocalDateTime_SP");
     private static String getObjectOffsetDateTimeProcedureName = RandomUtil.getIdentifier("CallableStatementTest_getObjectOffsetDateTime_SP");
     private static String procName = RandomUtil.getIdentifier("procedureTestCallableStatementSpPrepare");
+    private static String manyParamsTable = RandomUtil.getIdentifier("manyParam_Table");
```

Comment:
nitpick: why not add AbstractSQLGenerator.escapeIdentifier() here so we don't have to do this in multiple places?